bad-practice.js

```
@@ -0,0 +1,11 @@
+// This file is for testing CodeRabbit's code review
+const password = "123456";
+eval("console.log('This could be dangerous')");
+
+function add(a, b) {
+  return a + b;
+}
+
+if (true == 'true') {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)